opentelemetry-api/setup.py

```diff
@@ -44,7 +44,7 @@
     include_package_data=True,
     long_description=open("README.rst").read(),
     long_description_content_type="text/x-rst",
-    install_requires=["typing; python_version<'3.5'"],
+    install_requires=["typing; python_version<'3.5'", "aiocontextvars"],
```

Comment:
Is `python_version` missing for `aiocontextvars`?

Reply:
added it.